frontends/mit-open/src/pages/LearningPathListingPage/LearningPathListingPage.tsx

```diff
@@ -60,49 +68,27 @@ const EditListMenu: React.FC<EditListMenuProps> = ({ resource }) => {
   return (
     <SimpleMenu
       trigger={
-        <ActionButton
-          variant="text"
+        <StyledActionButton
+          variant="secondary"
+          edge="none"
           color="secondary"
           size="small"
+          mobile={isMobile}
           aria-label={`Edit list ${resource.title}`}
         >
           <MoreVertIcon fontSize="inherit" />
-        </ActionButton>
+        </StyledActionButton>
       }
       items={items}
     />
   )
 }
 
-type ListCardProps = {
-  list: LearningPathResource
-  onActivate: (resource: LearningPathResource) => void
-  canEdit: boolean
-}
-const ListCard: React.FC<ListCardProps> = ({ list, onActivate, canEdit }) => {
-  return (
-    <LearningResourceCardTemplate
-      variant="row-reverse"
-      resource={list}
-      imgConfig={imgConfigs["row-reverse-small"]}
-      footerActionSlot={canEdit ? <EditListMenu resource={list} /> : null}
-      onActivate={onActivate}
-    />
-  )
-}
-
 const LearningPathListingPage: React.FC = () => {
+  const isMobile = !useMuiBreakpointAtLeast("md")
```

Comment:
Mentioning: I have been avoiding using these. I'm not sure what the SSR situation is for `useMuiBreakpoint`. Where we can, I think CSS is a better solution, though I understand changing props via mobile vs desktop can't be done with CSS.

Reply:
It's not really needed here - I've replaced with css.